src/services/document-service.spec.js

```diff
@@ -1071,7 +1071,7 @@ describe('document-service', () => {
       });
     });
 
-    describe('when I search with a query that returns multiple documents', () => {
+    describe('when the query matches multiple documents', () => {
       it('does not contain archived documents', async () => {
         const results = await sut.getSearchableDocumentsMetadataByTags('music');
         expect(results.map(result => result.title)).not.toContain('Doc 4');
```

Comment:
We've got the test for searching "1 or 2" and the one searching "Doc 4", but I think we need one testing something like "Music 2", to cover the case of both tags matching in one single document, one matching exactly and one partially. This is a case not covered in the current tests as far as I could spot.

Reply:
True. I added a test case with two terms, one of them matching one doc partially, the other one matching a different doc exactly.